IEWebKitImpl/DOM.ts

```diff
@@ -0,0 +1,203 @@
+﻿//
+// Copyright (C) Microsoft. All rights reserved.
+//
+
+/// <reference path="Interfaces.d.ts"/>
+/// <reference path="Browser.ts"/>
+module F12.Proxy {
+    "use strict";
+
+    declare var browser: IBrowser;
+    export class DOMHandler {
+        private _mapUidToNode: Map<number, Node>;
+        private _mapNodeToUid: WeakMap<Node, number>;
+        private _nextAvailableUid: number;
+        private _windowExternal: any; //todo: Make an appropriate TS interface for external
+        private _elementHighlightColor: any;
+        constructor() {
+            this._mapUidToNode = new Map<number, Node>();
+            this._mapNodeToUid = new WeakMap<Node, number>();
+            this._nextAvailableUid = 2; // 1 is reserved for the root
+            this._windowExternal = (<any>external);
+            this._elementHighlightColor = {
+                margin: "rgba(250, 212, 107, 0.50)",
+                border: "rgba(120, 181, 51, 0.50)",
+                padding: "rgba(247, 163, 135, 0.50)",
+                content: "rgba(168, 221, 246, 0.50)"
+            };
+        }
+
+        private createChromeNodeFromIENode(node: Node): INode {
+            var inode: INode = {
+                nodeId: +this.getOrAssignUid(node),
```

Comment:
Originally I used it to convert strings to numbers, it is no longer needed so I'll remove it